website/docs/faqs/Troubleshooting/sl-alpn-error.md

```diff
@@ -0,0 +1,11 @@
+---
+title: I'm receiving an `Failed ALPN` error when trying to connect to the dbt Semantic Layer.
+description: "To resolve the 'Failed ALPN' error in the dbt Semantic Layer, create a SSL interception exception for the dbt Cloud domain."
+sidebar_label: 'Use SSL exception to resolve `Failed ALPN` error'
+---
+
+If you're receiving a `Failed ALPN` error when trying to connect the dbt Semantic Layer with the available [data integration tool](/docs/use-dbt-semantic-layer/avail-sl-integrations), it typically happens when connecting from a computer behind a corporate VPN or Proxy, such as Zscaler. 
```

Comment:
another product we experienced issues with was Check Point. Might be worth changing to `such as Zscaler or Check Point`